notesapplication/notes/api_v1/serializers.py

```diff
@@ -1,21 +1,57 @@
-from ast import Not
-from dataclasses import field
-from django.contrib.auth.password_validation import validate_password
 from rest_framework import serializers
 from rest_framework.validators import UniqueValidator
+from rest_framework.pagination import PageNumberPagination
+from django.core.paginator import Paginator
 from ..models import Note, NoteVersion, Comment
 from datetime import date
 
+NUMBER_TO_GET_LATEST_COMMENT = 1
+
+
+class CommentSerializer(serializers.ModelSerializer):
+    """
+    CommentSerializer : Serializer for comment class and all operations of it
+    """
+
+    class Meta:
+        model = Comment
+        fields = "__all__"
+
+    def create(self, validated_data):
+
+        request = self.context.get("request", None)
+        comment = Comment.objects.create(
+            text=validated_data["text"],
+            note_id=validated_data["note_id"],
+            user=request.user,
```

Comment:
how can we create a comment without a user assuming is `request.user is None`. There should be proper validations on DB level and error handling here if a user is mandatory in creating a comment.